Docs/nahmii-provider.md

```diff
@@ -33,26 +34,34 @@ A class providing low-level access to the _hubii nahmii_ APIs.
 **Kind**: Exported class  
 <a name="new_module_nahmii-sdk--NahmiiProvider_new"></a>
 
-#### new NahmiiProvider(nahmiiDomain, apiAppId, apiAppSecret, nodeUrl, network)
+#### new NahmiiProvider(nahmiiDomain, apiAppId, apiAppSecret, nodeUrl, network, [operator])
 Construct a new NahmiiProvider.
 Instead of using this constructor directly it is recommended that you use
 the NahmiiProvider.from() factory function.
 
 
-| Param | Type | Description |
-| --- | --- | --- |
-| nahmiiDomain | <code>string</code> | The domain name for the nahmii API |
-| apiAppId | <code>string</code> | nahmii API app-ID |
-| apiAppSecret | <code>string</code> | nahmii API app-secret |
-| nodeUrl | <code>string</code> | url to an ethereum node to connect to |
-| network | <code>string</code> \| <code>number</code> | a known ethereum network name or ID |
+| Param | Type | Default | Description |
+| --- | --- | --- | --- |
+| nahmiiDomain | <code>string</code> |  | The domain name for the nahmii API |
+| apiAppId | <code>string</code> |  | nahmii API app-ID |
+| apiAppSecret | <code>string</code> |  | nahmii API app-secret |
+| nodeUrl | <code>string</code> |  | url to an ethereum node to connect to |
+| network | <code>string</code> \| <code>number</code> |  | a known ethereum network name or ID |
+| [operator] | <code>EthereumAddress</code> | <code></code> | The address of the nahmii operator |
 
```

Comment:
Instead of having an empty column, would it be better to populate it with undefined where no default apply?

Reply:
Not sure what you are referring to?? This MD file is generated from the JSDOC comments in the code - should not be manually edited.

Reply:
Ah I see. the default value is "undocumented".

![image](https://user-images.githubusercontent.com/1760828/61776525-8453ed00-adfb-11e9-9324-7ab896ee96c3.png)

Reply:
Tried updating it in the JSDOC comment, but it seems the "=null" isnt picked up by the doc generator for some reason.